src/sentry/options/defaults.py

```diff
@@ -477,6 +477,13 @@
     default=[],
     flags=FLAG_ALLOW_EMPTY | FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
 )
+# Separate compute and IO.
+register(
```

Comment:
flagpole may be easier than options for an org based rollout, but either way is fine.